src/elements/element.line.js

```diff
@@ -198,6 +198,11 @@ function _getInterpolationMethod(options) {
 	return _pointInLine;
 }
 
+function coordsAnimated(point) {
+	const anims = point.$animations;
+	return anims && ((anims.x && anims.x.active) || (anims.y && anims.y.active));
```

Comment:
Was going to ask what happens if something else animate, eg radius, but then I realized that only `x` and `y` affect the path. Perhaps this is worth a comment?